.env.example

```diff
@@ -1,10 +1,29 @@
-EMAIL_USERNAME="youremail@gmail.com"
-EMAIL_PASSWORD="secretpassword"
-NOTIFICATION_TEST="false"
-PAGE_TIMEOUT="30000"
-RATE_LIMIT_TIMEOUT="5000"
-SLACK_CHANNEL="SlackChannelName"
-SLACK_TOKEN="slack-token"
-STORES="bestbuy,bandh,nvidia"
-PHONE_NUMBER="1234567890"
-PHONE_CARRIER="tmobile"
\ No newline at end of file
+// All configuration variables are OPTIONAL.  Uncomment any variables you wish to use or use none of them!
```

Comment:
I think this type of documentation should live inside the README. In my opinion, it's descriptive enough, but if we want extra help documentation, let's put it inside the wiki.

Reply:
Okay, we can remove it.  I just know a lot of people in discord are renaming the .env.example without modifying anything inside of the .env and they are getting lots of errors because they haven't filled in API information etc.